CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ set(LIBRARY_SOVERSION ${LIBRARY_VERSION_MAJOR})
 
 option(BUILD_SHARED_LIBS "Build shared avif library" ON)
 
-option(AVIF_ENABLE_WERROR "Treat all compiler warnings as errors" ON)
+option(AVIF_ENABLE_WERROR "Treat all compiler warnings as errors" OFF)
```

Comment:
Please mention this change in CHANGELOG.md. This is useful information to libavif package maintainers.

Let's also update appveyor.yml and the build.sh scripts in tests/.